apps/vscode-wing/language-configuration.json

```diff
@@ -3,7 +3,7 @@
         "lineComment": "//",
         "blockComment": [ "/*", "*/" ]
     },
-    "wordPattern": "/([a-zA-Z_$][A-Za-z_$0-9]*)/",
+    "wordPattern": "([a-zA-Z_@][\\w_]*)",
```

Comment:
purposefully not considering `$` a valid char anymore?

Reply:
I agree with this change. We should reserve `$` for JavaScript implementation details.

Reply:
That part of the change was unintentional, I completely forgot that `$` is a valid identifier character.
I'll undo the change here even though I think I agree because it would belong in another PR.